test/initial_draft.cpp

```diff
@@ -101,50 +100,53 @@ using segment_result_type =
     simplify this code by handing the multi-argument case earlier (somehow).
 */
 
+#define STLAB_FWD(x) std::forward<decltype(x)>(x)
```

Comment:
I don't know where this goes long term, but I feel strongly we need it somewhere. If I see another `std::forward<decltype(args)>(args)...` I may weep.

Reply:
Should probably be `CHAINS_FORWARD()` and would go in `<chains/utility.hpp>` (same header name that contains `std::forward<>` - not a general dumping ground of utilities).